README.md

```diff
@@ -1 +1,9 @@
 # cs-5666-scatt-gp
+
+## Demo Instructions
+* Login to the Appalachian student server.
+* Clone the SCATT repo with the command `git clone https://github.com/JaysGitLab/cs-5666-scatt-gp.git`.
+* Go into the SCATT directory with the command `cd cs-5666-scatt-gp`.
+* Enter `make scatt` and hit return.
+* Enter the path to your SCATT submissions directory when prompted.
```

Comment:
@haietza @KaraBeason How detailed should we make this? Are we going to allow them to test some of their own submissions they put in a directory, or ours from our project? If we are allowing their own, which is what I got from this line, I think we should have some directions on what files go into the directory and if the directory has to be in a certain place. I know there is some assumptions made, our client will have to know with the log-on to the student server, but I think it will be beneficial to have a more detailed instructions for first time clueless users.